pkg/instrumentation/helper.go

```diff
@@ -14,7 +14,23 @@
 
 package instrumentation
 
-import corev1 "k8s.io/api/core/v1"
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+func defaultInitContainerResourceRequirements() corev1.ResourceRequirements {
+	return corev1.ResourceRequirements{
+		Limits: corev1.ResourceList{
+			corev1.ResourceCPU:    resource.MustParse("500m"),
```

Comment:
I am curious why this is set to `500m`/0.5 CPU? Did you try lower numbers e.g. `100m`?

Reply:
y, even 50 works. Iam fine with 100 too. Its just the upper limit.